RecoLocalCalo/HcalRecProducers/plugins/alpaka/Mahi.dev.cc

```diff
@@ -0,0 +1,1470 @@
+                                                                   i,
+                                                                   0,
+                                                                   acc25nsVec,
+                                                                   diff25nsItvlVec,
+                                                                   accVarLenIdxMinusOneVec,
+                                                                   diffVarItvlIdxMinusOneVec,
+                                                                   accVarLenIdxZeroVec,
+                                                                   diffVarItvlIdxZeroVec);
+                      printf("pulse(%d) = %f\n", i, value);
+                    }
+                    printf("\n");
+                    for (int i = 0; i < hcal::constants::maxSamples; i++) {
+                      auto const value = compute_pulse_shape_value(t0p,
+                                                                   i,
+                                                                   0,
+                                                                   acc25nsVec,
+                                                                   diff25nsItvlVec,
+                                                                   accVarLenIdxMinusOneVec,
+                                                                   diffVarItvlIdxMinusOneVec,
+                                                                   accVarLenIdxZeroVec,
+                                                                   diffVarItvlIdxZeroVec);
+                      printf("pulseP(%d) = %f\n", i, value);
+                    }
+                    printf("\n");
+                    for (int i = 0; i < hcal::constants::maxSamples; i++) {
+                      auto const value = compute_pulse_shape_value(t0m,
+                                                                   i,
+                                                                   0,
+                                                                   acc25nsVec,
+                                                                   diff25nsItvlVec,
+                                                                   accVarLenIdxMinusOneVec,
+                                                                   diffVarItvlIdxMinusOneVec,
+                                                                   accVarLenIdxZeroVec,
+                                                                   diffVarItvlIdxZeroVec);
+                      printf("pulseM(%d) = %f\n", i, value);
+                    }
+                  }
+#endif
+
+                  // FIXME: shift should be treated properly,
+                  // here assume 8 time slices and 8 samples
+                  auto const shift = 4 - soi;  // as in cpu version!
+
+                  // auto const offset = ipulse - soi;
+                  // auto const idx = sample - offset;
```

Comment:
I think you can remove these comments: they have been sitting in the CUDA code for the past 4 years, and nobody knows why...

Reply:
```suggestion
```

Reply:
Done